primeqa/pipelines/components/base.py

```diff
@@ -89,5 +102,18 @@ def __hash__(self) -> int:
         raise NotImplementedError
 
     @abstractmethod
-    def retrieve(self, input_texts: List[str], *args, **kwargs):
+    def retrieve(self, queries: List[str], *args, **kwargs):
```

Comment:
If we are changing signature of `retrieve` method, we need to follow that signature change in both `sparse` and `dense` retrievers.